state_machine/applications/flight/lib/logs.py

```diff
@@ -1,48 +1,55 @@
 from pycubed import cubesat
-from state_machine import state_machine
 import struct
 try:
     from ulab.numpy import array
 except ImportError:
     from numpy import array
 
 
-beacon_format = 'b' + 'f' * 11  # 1 char + 11 floats
+beacon_format = 'H' + 'f' * 11  # 1 short + 11 floats
 
-def beacon_packet(task):
-    """Creates a beacon packet containing the: CPU temp, IMU temp, gyro, acceleration, magnetic, and state byte.
-    The state byte is the index of the current state in the alphabetically ordered state list.
-    This data is packed into a c struct using `struct.pack`.
+def beacon_packet():
+    """Creates a beacon packet containing the: boot count, battery voltage,
+    CPU temperature, IMU temperature, gyro reading, mag reading,
+    radio signal strength (RSSI), radio frequency error (FEI).
 
-    If no IMU is attached it returns a packet of 0s.
+    This data is packed into a c struct using `struct.pack`.
     """
-    if not cubesat.imu:
-        task.debug('IMU not initialized')
-        return bytes([0, 0, 0, 0, 0])
 
+    boot_count = cubesat.c_boot
+    vbatt = cubesat.battery_voltage
     cpu_temp = cubesat.temperature_cpu
     imu_temp = cubesat.temperature_imu
     gyro = cubesat.gyro
-    acc = cubesat.acceleration
     mag = cubesat.magnetic
-    state_byte = state_machine.states.index(state_machine.state)
```

Comment:
I'd like to keep the state byte, it will tell us if we're out of deployment mode and stuff like that.